snapshots/gapminder/2023-09-22/total_fertility_rate.xlsx.dvc

```diff
@@ -0,0 +1,37 @@
+# Learn more at:
+# http://docs.owid.io/projects/etl/en/latest/architecture/metadata/reference/origin/
+meta:
+  origin:
+    # Data product / Snapshot
+    title: Babies per woman (total fertility rate)
+    description: |-
+      Sources
+      — 1800 to 1950 (and in some cases also years after 1950): Gapminder v6 which were compiled and documented by Mattias Lindgren, see previous versions further down on this page.
+
+      — 1950 to 2014: In most cases we use the latest UN estimates from World Population Prospects 2017 published in the file with Annually interpolated demographic indicators, called WPP2017_INT_F01_ANNUAL_DEMOGRAPHIC_INDICATORS.xlsx , accessed on September 2, 2017.
+
+      — 2015 – 2099: We use the UN forecast of future fertility rate in all countries, called median fertility variant.
```

Comment:
is UN WPP 2022 not suitable for this?

Reply:
Hmm good point, it probably would be best to use UN WPP from 1950 onwards.

Reply:
I've added this in now, the metadata doesn't seem to include UN WPP though. Is currently better to just manually curate the metadata for cases like this?